charts/div-ccd-definitions/values.preview.template.yaml

```diff
@@ -58,6 +58,7 @@ ccd:
         - divorce_as_caseworker_superuser@mailinator.com|DIVORCE|DIVORCE|Submitted
         - divorce_as_caseworker_la@mailinator.com|DIVORCE|DIVORCE|Submitted
         - divorce_as_caseworker_admin@mailinator.com|DIVORCE|DIVORCE|Submitted
+        - divorce_respondent_solicitor@mailinator.com|DIVORCE|DIVORCE|Submitted
```

Comment:
what does it do?

Reply:
It adds an extra solicitor user - we need a respondent solicitor for testing

Reply:
ok